openfe/setup/atom_mapping/perses_mapper.py

```diff
@@ -50,24 +51,29 @@ def __init__(self, allow_ring_breaking: bool = True,
         self.coordinate_tolerance = coordinate_tolerance
 
     def _mappings_generator(self, molA, molB):
+        # Construct Perses Mapper
         _atom_mapper = AtomMapper(
             use_positions=self.use_positions,
             coordinate_tolerance=self.coordinate_tolerance,
             allow_ring_breaking=self.allow_ring_breaking)
 
-        # Type of mapping
+        # Try generating a mapping
         try:
             _atom_mappings = _atom_mapper.get_all_mappings(
                     old_mol=molA.to_openff(), new_mol=molB.to_openff())
         except InvalidMappingException:
-            _atom_mappings = []
+            return
+
+        # Catch empty mappings here
+        if (_atom_mappings is None):
```

Comment:
```suggestion
        if _atom_mappings is None:
```
Odd @dwhswenson's edit didn't filter through.